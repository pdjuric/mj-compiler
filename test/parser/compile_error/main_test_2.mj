program main_test
{
    int foo() { }
    void bar() { }
}