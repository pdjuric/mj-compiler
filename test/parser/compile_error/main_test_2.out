ERROR 17:08:03,135 - line   1 - Function void main() is not defined.